### PR TITLE
Add updateAt field (closes #27)

### LINKED ISSUE #27: Add UpdateAt field
UpdatedAt field in every foulkon entity (user, group and policy).

### PR DESCRIPTION
**Pull request by rsoletob**

### CODE REVIEW

File: database/postgresql/user.go [hunk 1]
```diff
@@ -111,24 +112,19 @@ func (u PostgresRepo) GetUsersFiltered(filter *api.Filter) ([]api.User, int, err
...
 		ExternalID: user.ExternalID,
 		Path:       user.Path,
 		CreateAt:   user.CreateAt.UnixNano(),
+		UpdateAt:   user.UpdateAt.UnixNano(),
 		Urn:        user.Urn,
 	}
 
 	// Update user
-	query := u.Dbmap.Model(&userDB).Update(userUpdated)
+	query := u.Dbmap.Model(&User{ID: user.ID}).Update(userDB)
```
Comment (gerson24) [inline database/postgresql/user.go]: Updates?

File: database/postgresql/group_test.go [hunk 2]
```diff
@@ -676,29 +697,47 @@ func TestPostgresRepo_UpdateGroup(t *testing.T) {
 func TestPostgresRepo_RemoveGroup(t *testing.T) {
```
Comment (gerson24) [inline database/postgresql/group_test.go]: should be like `TestPostgresRepo_RemoveGroup`

File: database/postgresql/policy_test.go [hunk 3]
```diff
@@ -581,37 +571,38 @@ func TestPostgresRepo_UpdatePolicy(t *testing.T) {
 func TestPostgresRepo_RemovePolicy(t *testing.T) {
```
Comment (gerson24) [inline database/postgresql/policy_test.go]: should be like `TestPostgresRepo_RemoveGroup`

Comment (rsoletob) [inline database/postgresql/policy_test.go]: Updated but with some additional logic

File: database/postgresql/group_test.go (hunk 2 above)
Comment (rsoletob) [inline database/postgresql/group_test.go]: Updated but with some additional logic

File: database/postgresql/user.go (hunk 1 above)
Comment (rsoletob) [inline database/postgresql/user.go]: Yes, thanks

File: api/policy.go [hunk 4]
```diff
@@ -278,59 +279,73 @@ func (api AuthAPI) UpdatePolicy(requestInfo RequestInfo, org string, policyName
...
 	}
+
 	if err != nil {
-		if apiError := err.(*Error); apiError.Code == UNAUTHORIZED_RESOURCES_ERROR || apiError.Code == UNKNOWN_API_ERROR {
+		if apiError := err.(*Error); apiError.Code != POLICY_BY_ORG_AND_NAME_NOT_FOUND {
 			return nil, err
 		}
 	}
 
 	// Get Policy Updated
```
Comment (rsoletob) [inline api/policy.go]: done

File: api/group.go [hunk 5]
```diff
@@ -286,29 +286,41 @@ func (api AuthAPI) UpdateGroup(requestInfo RequestInfo, org string, name string,
 	}
 
 	if err != nil {
-		if apiError := err.(*Error); apiError.Code == UNAUTHORIZED_RESOURCES_ERROR || apiError.Code == UNKNOWN_API_ERROR {
+		if apiError := err.(*Error); apiError.Code != GROUP_BY_ORG_AND_NAME_NOT_FOUND {
 			return nil, err
 		}
 	}
 
 	// Get Group updated
```
Comment (rsoletob) [inline api/group.go]: done

**Pull request merged**